src/static/bitbucket/index.js

```diff
@@ -0,0 +1,155 @@
+const endpoint = window.location.origin;
+
+const landButtonView = pullRequestId => {
+  return `<div>
+    <p>This PR is not queued for Landing yet, click "Land" below or <a href="google.com">here</a> for more information</p>
+    <br>
+    <button type="button" class="ak-button ak-button__appearance-primary" onClick="landButtonClicked(${
+      pullRequestId
+    })">
+      Land this PR
+      </button>
+  </div>`;
+};
+
+const queueView = positionInQueue => {
+  return `<div>
+    <p>This PR is queued for release now. See <a href="goog.e.com">here</a> for more information about what this means</p>
+    <p>Position in queue: <span class="numberInQueue">${positionInQueue +
+      1}</span><p>
+    <p>
+      <button type="button" class="ak-button ak-button__appearance-default" onClick="cancelButtonClicked()">
+        Cancel release
+      </button>
+    </p>
+  </div>`;
+};
+
+const cancellingView = () => {
+  return `<div>
+    <p>Cancelling...</p>
+  </div>`;
+};
+
+const cancelledView = () => {
+  return `<div>
+    <p>Release has been cancelled.</p>
+    <p>Please refresh page to re-submit</p>
+  </div>`;
+};
+
+const landingView = () => {
+  return `<div>
+    <p>Sending land request...</p>
+  </div>`;
+};
+
+const errorCreatingLandRequestView = err => {
+  console.error(err);
+  const reason = err.reason || "We honestly don't know... See error console";
+  return `<div>
+    <p>There was an error whilst queueing your land request:</p>
+    <p style="color: red">${reason}</p>
+  </div>`;
+};
+
+function getCurrentQueue() {
+  return fetch(`${endpoint}/api/current-queue`)
+    .then(resp => resp.json())
+    .catch(err => console.error('error ', err));
+}
+
+// Fetches the user, repo and id vars
+function getQueryStringVars() {
+  const queryString = window.location.search.substring(1);
+  return window.Qs.parse(queryString);
+}
+
+function landButtonClicked() {
+  setView(landingView());
+
+  const queryStringVars = getQueryStringVars();
+  const pullRequestId = queryStringVars.pullRequestId;
```

Comment:
Pretty terrible because I didnt want to use destructuring without babel here, so would be a lot cleaner if we used a bundler or even just babel